src/main/java/org/contextmapper/standalone/cli/CommandLineInput.java

```diff
@@ -0,0 +1,50 @@
+package org.contextmapper.standalone.cli;
```

Comment:
Would be nice if you could add the Context Mapper copyright as we do in all Java classes:

/*
 * Copyright 2020 The Context Mapper Project Team
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */